public/images/external-link.svg

```diff
@@ -0,0 +1,8 @@
+<svg width="284px" height="284px" viewBox="0 0 284 284" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
[We recently had to change the folder of icons](https://github.com/greenpeace/planet4-master-theme/commit/1453107e0ef8f691b25e2fa440d3a68707a0c09c), because somehow going up `../../..` caused a weird webpack build error, so we had to put it closer to the stylesheet folder. They are now in `assets/src/images/icons`.

Can you move these there too? :pray: